x-pack/plugins/maps/public/connected_components/mb_map/mb_map.tsx

```diff
@@ -150,7 +150,7 @@ export class MbMap extends Component<Props, State> {
     }
   }, 256);
 
-  _getMapState() {
+  _getMapExtentState() {
```

Comment:
can we type as `MapExtentState`, and reuse where appropriate (see higher comment)